dev/breeze/src/airflow_breeze/commands/custom_param_types.py

```diff
@@ -53,6 +54,17 @@ def get_metavar(self, param) -> str:
         return f"[{choices_str}]"
 
 
+class AnswerChoice(BetterChoice):
+    """
+    Stores forced answer if it has been selected
+    """
+
+    name = "AnswerChoice"
+
+    def convert(self, value, param, ctx):
+        set_forced_answer(value)
```

Comment:
This allowed us to remove manual "set_forced_answer" wherever `--answer` was used :)